pkg/static/login.js

```diff
@@ -889,6 +889,8 @@
                 }
             } else if (xhr.status == 403) {
                 login_failure(_(decodeURIComponent(xhr.statusText)) || _("Permission denied"));
+            } else if (xhr.status == 500) {
```

Comment:
in all other cases we `decodeURIComponent()`. Don't we need it here as well?

Reply:
So this change broke a unit test and just fixes one issue out of multiple issues with our login page in cockpit client. I've dropped this change as I think we need a better approach here.

Reply:
why have you dropped this commit?